backend/coins/eth/types/types.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2018 Shift Devices AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package types
```

Comment:
We don't have `btc/types` package either…

Reply:
So...?

This exists because I reference it from the etherscan package. If it was in the eth package, it would be an import cycle. Didn't feel like it belonged into the etherscan package.